fbw-common/src/systems/shared/src/publishers/AdiruBusPublisher.ts

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) 2024 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
+
+import {
+  EventBus,
+  IndexedEventType,
+  PublishPacer,
+  SimVarPublisher,
+  SimVarPublisherEntry,
+  SimVarValueType,
+} from '@microsoft/msfs-sdk';
+
+interface AdiruBusBaseEvents {
+  /** Raw Arinc 429 word. */
+  a32nx_adiru_true_track: number;
```

Comment:
```suggestion
  a32nx_ir_true_track: number;
```
IR and ADR have separate bus outputs. The publishers should be arranged around output busses. Thew JSDoc should also specify what data is contained within.

Reply:
Ok, I renamed the file to IrBusPublisher.ts and changed everything accordingly. Didn't all of the IR outputs though, only the one I need for now. The PR is massive as it is right now...